ui/plugin-system/src/runtime/TimeRangeProvider/TimeRangeProvider.tsx

```diff
@@ -12,11 +12,18 @@
 // limitations under the License.
 
 import React, { useMemo, useState, useCallback, createContext, useContext } from 'react';
-import { AbsoluteTimeRange, isRelativeTimeRange, TimeRangeValue, toAbsoluteTimeRange } from '@perses-dev/core';
-import { useSetTimeRangeParams } from './query-params';
+import {
+  AbsoluteTimeRange,
+  isRelativeTimeRange,
+  RelativeTimeRange,
+  TimeRangeValue,
+  toAbsoluteTimeRange,
+} from '@perses-dev/core';
+import { useSetRefreshIntervalParams, useSetTimeRangeParams } from './query-params';
 
 export interface TimeRangeProviderProps {
   initialTimeRange: TimeRangeValue;
+  initialRefreshInterval?: RelativeTimeRange;
```

Comment:
I'm thinking if this can be simplified to just using `DurationString`?

Reply:
Yeah good call, this is better than what I suggested before

Reply:
Changed!
Can you take a look @eunicorn @sjcobb